src/frontend/screens/Library/components/InstallModal/SideloadDialog/index.tsx

```diff
@@ -102,6 +102,22 @@ export default function SideloadDialog({
     setWine()
   }, [title])
 
+  useEffect(() => {
+    setTimeout(async () => {
+      try {
+        const res = await fetch(
+          `https://steamgrid.usebottles.com/api/search/${title}`
```

Comment:
Adding a link to the documentation of this API might be helpful for the future

Reply:
Yes, Mirko just sent a message to me on Discord saying that we could use it. Although I think we should host our own in the future.